gateway/Cargo.toml

```
@@ -37,6 +37,13 @@ metrics = { default-features = false, features = ["std"], optional = true, versi
 simd-json = { default-features = false, features = ["serde_impl", "swar-number-parsing"], optional = true, version = "0.4" }
 tracing = { default-features = false, features = ["std", "attributes"], optional = true, version = "0.1" }
 
+# TLS libraries
+# They are needed to track what is used in tokio-tungstenite
+native-tls = { default-features = false, optional = true, version = "0.2.8" }
+rustls-tls = { default-features = false, optional = true, package = "rustls", version = "0.20" }
+rustls-native-certs = {default-features = false, optional = true, version = "0.6" }
+webpki-roots = {default-features = false, optional = true, version = "0.22" }
```

tiny styling nit that might've been overseen in the last suggestion diff

```suggestion
rustls-native-certs = { default-features = false, optional = true, version = "0.6" }
webpki-roots = { default-features = false, optional = true, version = "0.22" }
```